app/components/provider_interface/application_timeline_component.rb

```diff
@@ -50,8 +50,24 @@ def note_events
       end
     end
 
+    def feedback_events
+      if application_choice.rejected_by_default
+        application_choice.audits.where(action: 'update').where(
+          'audited_changes ? \'reject_by_default_feedback_sent_at\'',
```

Comment:
Can we do something like `'audited_changes ? :key', key: :reject_by_default_feedback_sent_at` to avoid escaping quotes here?

Reply:
Is the `?` operator here a shorthand for a json function like `jsonb_exists`? 
It would be clearer to use the underlying function name if so.

Reply:
No, it's a (jsonb-specific) built-in operator (see https://www.postgresql.org/docs/13/functions-json.html#FUNCTIONS-JSONB-OP-TABLE). I've added a comment to the code explaining it.

Reply:
Understood, I was leaning towards using the longhand for this operator (though as you point out it's well documented):

`SELECT oprname, oprcode FROM pg_operator WHERE oprname = '?'` which returns `jsonb_exists`

So this could also be written as `.where("jsonb_exists('audited_changes', 'reject_by_default_feedback_sent_at'))"`
Not a blocker, just a style preference.

Reply:
Ok, combining both your comments into:

```
        application_choice.audits.where(action: 'update').where(
          'jsonb_exists(audited_changes, :key)',
          key: :reject_by_default_feedback_sent_at,
        )
```